panel/lxqtpanel.cpp

```diff
@@ -640,7 +635,8 @@ void LxQtPanel::showAddPluginDialog()
 void LxQtPanel::addPlugin(const LxQt::PluginInfo &desktopFile)
 {
     QString settingsGroup = findNewPluginSettingsGroup(desktopFile.id());
-    loadPlugin(desktopFile, settingsGroup);
+    if (0 != loadPlugin(desktopFile, settingsGroup))
```

Comment:
It's there to leave previous functionality untouched (if loading of newly added plugin failed, nothing was added into *mPlugins* => no new plugin was added into settings). IMO this is correct behaviour.